content/foundations/physical_properties/physical_properties_dielectric_permittivity.rst

```diff
@@ -15,7 +15,7 @@ Dielectric permittivity is in fact a complex value, often written \\( \\epsilon_
 
 The so-called "real" part, \\(\\epsilon_r^{\\prime}\\), is the *relative dielectric constant*\\( \\), often introduced in electronics or physics courses in the context of capacitors. It is a storage component measured as capacitance per unit length. (Capacitance is "the amount of charge a material can hold" for a given applied voltage.) At different frequencies, polarization occurs at different scales: at very high frequencies, only subatomic particles can be polarized. At GPR frequencies, the reorientation of dipolar molecules is the largest contribution, hence water's importance in determining the velocity of EM waves in a material. Note that \\(\\epsilon_R = 80\\) for water, whereas \\(\\epsilon_R < 10\\) for most other common materials.
 
- .. figure:: ../GPR/images/dielectric_responses.jpg
+ .. figure:: ./images/images_duplicates/dielectric_responses.jpg
```

Comment:
I agree we need to find a more elegant solution.
Duplicating images is mostly due to the choice to include it in the foundation chapter. To be consistent, I moved the physical properties folder in the foundation folder. Some links were referring to image stored in other folders. But as the physical properties went "one section below", the "../" command was not referring anymore to the content folder but to "foundation". That's why I had to relink images.By duplicating image, in the case we want to move again the physical properties folder elsewhere, we will have not to do this again.
(let me know if this explanation is too fuzzy...)